addons/teamColors/functions/fnc_setColor.sqf

```diff
@@ -15,8 +16,8 @@
 
 TRACE_1("Params",_this);
 
-[{
-    params ["_unit", "_fireteam"];
-    TRACE_2("setting fireteam",_unit,_fireteam);
-    ["CBA_teamColorChanged", [_unit, _fireteam], [_unit]] call CBA_fnc_targetEvent;
-}, _this, 1] call CBA_fnc_waitAndExecute;
+[
+    {(_this select 0) assignTeam (_this select 1);},
+    {!isNull player},
```

Comment:
I think the order is reversed, `[condition,statement,args]`
But this also runs on dedi, so player will be null there